app/controllers/api/v1/cars_controller.rb

```diff
@@ -1,12 +1,40 @@
 class Api::V1::CarsController < ApplicationController
   def index
     @cars = Car.all
-    render json: @cars
+    if @cars.empty?
+      render json: { message: 'No cars found' }
+    else
+      render json: @cars
+    end
```

Comment:
You could add responses status codes here, to make API consumption easy and straightforward.